Automattic/wp-calypso
# Add/support page block2

#### Proposed Changes

* Adds Gutenberg block to embed support page according to the design outlined in 78-gh/Automattic/lighthouse-forums

Things that will be implemented in a separate PR:
- Display the number of people that find the page useful
- Auto-creating the block on pasting the URL inside the editor
- Block loading state
- Transformations

#### Testing Instructions

* Run `yarn dev --sync` to sync code to your sandbox
* On your sandbox, edit `wp-content/mu-plugins/wpcom-bbpress.php` and add these lines
```
		if ( file_exists( WP_CONTENT_DIR . '/a8c-plugins/happy-blocks/index.php' ) ) {
			require_once WP_CONTENT_DIR . '/a8c-plugins/happy-blocks/index.php';
		}
```
to the function `wpcom_bbpress_load_plugins` to load `happy-blocks` plugin.
* Create a new topic, in the editor press `/` and search for "WordPress Guide" block.
* Insert support page URL, for example, https://wordpress.com/support/domains/change-a-domain/ and verify its content has been embedded in the editor.
* Save the topic, and verify embedded block was rendered in the topic view:

Note, that after merging PR D90725-code it should be possible to install it using Calypso 'install-plugin.sh`

#### Pre-merge Checklist
